PEMPal.1.0/Views/HomePageView.swift

```diff
@@ -38,6 +38,7 @@ struct HomePageView: View {
             .padding(.top, 50)
             .padding(.trailing, 200)
             
+            PEMStatus()
             lastDay(globalModel: globalModel)
             PEMButton(globalModel: globalModel)
```

Comment:
In the current form, since you're initializing a new instance of globalModel, there isn't a good way for the boolean to change when you press the Report PEM button (intuitively the button should turn to red every time you press the button)